modules/api/src/main/java/org/apache/fluo/api/data/Bytes.java

```diff
@@ -355,6 +354,36 @@ public static final Bytes of(String s, Charset c) {
   }
 
   /**
+   * Checks if this has the passed prefix
+   * 
+   * @param prefix
+   * @return true or false
```

Comment:
Would be useful to add a `@since 1.1.0` javadoc tag to document when the new API method first appeared.